src/rv32_template.c

```diff
@@ -69,10 +73,18 @@
  * |                                | using the memory address calculated as |
  * |                                | the sum of the src and the specified   |
  * |                                | offset.                                |
+ * | opt_ld, dst, field             | Map VM register to host register, and  |
```

Comment:
Apply the following refinement:
* Change from `opt_ld` to `ld`
* Change from `opt_ld_2` to `ld2`
* Change from `opt_ld_2_sext` `ld2s`

Reply:
However, the `ld` already exists; it serves another purpose. maybe we can rename it to `ra_ld`, `ra_ld2` and `ra_ld2s`

Reply:
Avoid using underscore (`_`) for mnemonics. Simply use something like `rald` or `vld`.